lib/wallet/http.js

```diff
@@ -973,7 +977,7 @@ class HTTP extends Server {
         return res.json(200, ns.getJSON(height, network));
       }
 
-      return res.json(404, {success: true});
+      return notFound();
```

Comment:
Why did this originally have success: true?

Reply:
It was just random JSON to parse, as far as I can tell. Nothing appears to expect that result, the only thing that calls it tries to extract a `message` field if the status code is non-200